### ISSUE TITLE
Fix Broken VoiceOver Bug

### ISSUE THREAD

**Issue by Sam2much96** [Sam2much96/RedditVideoGenerator#5]

**Comment by Sam2much96** [comment]
Bugs Details

(1) Voiceover script tts fails on 3rd Comment.
(2) If the post title has sub dialogue, it is completely ignored by the program, which limits the amount of subreddits that can be converted to content.

**Comment by Sam2much96** [comment]
Fixed by increasing the Total Duration constant, thus allowing the __create_voiceover() method to return an AudioClip() object rather than a None Object based on some nested code logic.
